packages/fondue/src/components/Modal/ModalFooter.tsx

```diff
@@ -1,12 +1,15 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { type ReactElement, useContext } from 'react';
+import { useContext, type ReactElement } from 'react';
 
 import { Button, ButtonSize } from '@components/Button';
 
 import { ModalLayout } from './context/ModalLayout';
 import { type ModalFooterProps } from './types';
 
+/**
+ * @deprecated Use `DialogFooter` from `@frontify/fondue/components` instead. Also check {@link https://github.com/Frontify/fondue/blob/main/packages/components/MIGRATING.md#dialog the migration guide}.
```

Comment:
Missing the dot between Dialog and Footer (same for the other files: Body, Footer, Header and Visual)

Reply:
added